engine/apps/public_api/serializers/escalation_policies.py

```diff
@@ -280,6 +295,11 @@ def update(self, instance, validated_data):
                     EscalationPolicy.STEP_NOTIFY_MULTIPLE_USERS_IMPORTANT,
                 ]:
                     instance.notify_to_users_queue.clear()
+                if step not in [
+                    EscalationPolicy.STEP_NOTIFY_TEAM_MEMBERS,
+                    EscalationPolicy.STEP_NOTIFY_TEAM_MEMBERS_IMPORTANT
+                ]:
+                    instance.notify_to_team_members = None
```

Comment:
Would you add some test(s) for the changes here?

Reply:
Added in `test_create_escalation_policy_using_notify_team_members`